src/tests/Interop/COM/NativeServer/Servers.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "stdafx.h"
 #include "Servers.h"
+#include <stdio.h>
```

Comment:
Is this needed?

Reply:
Leftover from printf debugging. Not sure why I added it here.